react-native/react/native/remote-manager.desktop.js

```diff
@@ -91,7 +91,6 @@ class RemoteManager extends Component {
             onRemoteClose={() => this.props.onCloseFromHeader(username)}
             component='tracker'
             username={username}
-            substore='tracker'
```

Comment:
This was really tricky actually, since I was using state from the config substore and I had forgotten about this. I have another PR that lets us efficiently think about the whole store by just sending diffs.